tests/RobotsTxtTest.php

```diff
@@ -110,4 +110,20 @@ public function it_can_handle_query_strings()
         $this->assertFalse($robots->allows('/en/admin?print'));
         $this->assertFalse($robots->allows('/en/admin?print=true'));
     }
+
+    /** @test */
+    public function test_allowed_link_for_title_case_custom_user_agent()
+    {
+        $robots = RobotsTxt::readFrom(__DIR__.'/data/robots.txt');
+
+        $this->assertTrue($robots->allows('/', 'UserAgent007'));
```

Comment:
So, with the changes in this PR, the user agent isn't case sensitive anymore right?

Let's test the also by adding

```
$this->assertTrue($robots->allows('/', strtolower('UserAgent007')));
```

Reply:
> So, with the changes in this PR, the user agent isn't case sensitive anymore right?
Yes. UserAgent007, useragent007, etc. are the same now.

I've added two more assertions for testing the lowercase versions of the string.